theme/scripts/emailSubscribe.ts

```diff
@@ -84,20 +86,22 @@ export default (function() {
 
     resetEmailSubscribe(emailSubscribe: HTMLElement) {
       const emailSubscribeInput: HTMLInputElement = emailSubscribe.querySelector(EmailSubscribe.SELECTORS.input);
-      const emailSubscribeSubmit: HTMLInputElement = emailSubscribe.querySelector(EmailSubscribe.SELECTORS.submit);
+      const emailSubscribeSubmitIcon: HTMLElement = emailSubscribe.querySelector(EmailSubscribe.SELECTORS.submitIcon);
 
       emailSubscribeInput.value = "";
       emailSubscribeInput.placeholder = EmailSubscribe.CONSTANTS.defaultPlaceholder;
 
-      emailSubscribeSubmit.value = EmailSubscribe.CONSTANTS.submitText;
-
       emailSubscribe.removeAttribute(EmailSubscribe.ATTRIBUTES.emailSubscribeState);
+
+      emailSubscribeSubmitIcon.innerHTML = EmailSubscribe.CONSTANTS.submitIcon;
     },
 
     startLoading(emailSubscribe: HTMLElement) {
       const emailSubscribeSubmit: HTMLInputElement = emailSubscribe.querySelector(EmailSubscribe.SELECTORS.submit);
+      const emailSubscribeSubmitIcon: HTMLElement = emailSubscribe.querySelector(EmailSubscribe.SELECTORS.submitIcon);
 
       emailSubscribeSubmit.value = EmailSubscribe.CONSTANTS.loading;
+      emailSubscribeSubmitIcon.innerHTML = "";
```

Comment:
Changing the innerHTML of the icon span because when the loading `...` appear the pencil icon should not be there